cli/tests/error_011_bad_module_specifier.ts.out

```diff
@@ -1,4 +1,4 @@
-[WILDCARD]error: Uncaught ImportPrefixMissing: relative import path "bad-module.ts" not prefixed with / or ./ or ../
+[WILDCARD]error: Uncaught ImportPrefixMissing: relative import path "bad-module.ts" not prefixed with / or ./ or ../. Imported from [WILDCARD]/error_011_bad_module_specifier.ts
```

Comment:
Does this may expose homeDir without `--allow-env` flag?

Reply:
It will print full path to file just like any other error. So yes, it does "expose" home dir, but Deno dies on errors anyway